DOC: Replace wikipedia links (#4853)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/users/index.md

| id        | `id`, `login`, or `login shortname` (as long as it is unambiguous) | URL        | String   | TRUE     |
| expand    | Valid value: `block`. If this parameter is specified, then account block details are included in the `_embedded` attribute. The [embedded object](/docs/reference/api/users/#user-block-object) lists information about how the account is blocked from access. | Query        | String   | FALSE     |

> > When fetching a user by `login` or `login shortname`, you should [URL encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) the request parameter to ensure special characters are escaped properly.  Logins with a `/` or `?`  character can only be fetched by `id` due to URL issues with escaping the `/` and `?` characters.

Reviewer: ```suggestion
> When fetching a user by `login` or `login shortname`, you should [URL encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) the request parameter to ensure that special characters are escaped properly. Usernames with a `/` or `?`  character can only be fetched by `id` due to URL issues with escaping the `/` and `?` characters.
```

Fetches a specific user when you know the user's `login`

> When fetching a user by `login`, [URL encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) the request parameter to ensure special characters are escaped properly.

Reviewer: ```suggestion
When fetching a user by `login`, [URL encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) the request parameter to ensure that special characters are escaped properly.
```

Fetches a specific user when you know the user's `login shortname` and the shortname is unique within the organization

> When fetching a user by `login shortname`, [URL encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) the request parameter to ensure special characters are escaped properly.

Reviewer: ```suggestion
When fetching a user by `login shortname`, [URL encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) the request parameter to ensure that special characters are escaped properly.
```
